config/rubocop/.rubocop.yml

```diff
@@ -512,7 +512,7 @@ Style/CommandLiteral:
 Style/CommentAnnotation:
   Description: >-
                  Checks formatting of special comments
-                 (TODO, FIXME, OPTIMIZE, HACK, REVIEW).
+                 (T-ODO, F-IXME, O-PTIMIZE, H-ACK, R-EVIEW).
```

Comment:
Sure either of those are fine by me.

I can go with `Checks formatting of annotation comments.`.

Personally, I prefer examples but I'm a newbie. ;)